src/commands/core/asset/create.ts

```
@@ -22,206 +22,341 @@
+  private async handleWizardMetadata(umi: any, wizard: any, imageUri?: string) {
+    // Create new metadata from wizard answers
+    let animationUri: string | undefined
+
+    // Upload image file (required)
+    const imageSpinner = ora('Uploading image...').start()
+    const imageResult = await uploadFile(umi, wizard.image).catch((err) => {
+      imageSpinner.fail(`Failed to upload image. ${err}`)
+      throw err
+    })
+    imageSpinner.succeed(`Image uploaded to ${imageResult.uri}`)
+    imageUri = imageResult.uri
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)